ci/plan.yml

```diff
@@ -241,9 +247,14 @@ SUITES:
             soft_fail: true # performance regression
           tf-dnnl-cpu:
             conda_env: ci/conda/tensorflow-dnnl.yml
+        skip_platforms:
+          - pml-vk-gen
       resnet50:
         precision: high
+        perf_threshold: 0.001 # ignore perf_fail for now
```

Comment:
Can we mark this on a platform basis? Not all platforms will perform this badly.